packages/extension-ui/src/Popup/Signing/Request/SignArea.tsx

```diff
@@ -111,27 +107,33 @@ function SignArea({ buttonText, className, error, isExternal, isFirst, isLast, s
   };
 
   return (
-    <form onSubmit={onSubmit}>
-      <div className={className}>
-        {isFirst && !isExternal && (
-          <>
-            {isLocked && (
-              <Unlock
-                error={error}
-                isBusy={isBusy}
-                onSign={_onSign}
-                password={password}
-                setError={setError}
-                setPassword={setPassword}
-              />
-            )}
-            <RememberPasswordCheckbox />
-          </>
-        )}
-      </div>
-      <CustomButtonArea>
+    <>
+      <form
+        id={formId}
+        onSubmit={onSubmit}
+      >
+        <div className={className}>
+          {isFirst && !isExternal && (
+            <>
+              {isLocked && (
+                <Unlock
+                  error={error}
+                  isBusy={isBusy}
+                  onSign={_onSign}
+                  password={password}
+                  setError={setError}
+                  setPassword={setPassword}
+                />
+              )}
+              <RememberPasswordCheckbox />
+            </>
+          )}
+        </div>
+      </form>
+      <ButtonArea>
```

Comment:
I think buttons could use `margin-inline: -16px`
![image](https://github.com/Cardinal-Cryptography/aleph-zero-signer/assets/35614528/3cccd84b-351c-435a-9602-aba4d867bfaa)

Reply:
Actually the `<BottomWrapper>` needs it and the buttons themselves need `+16px`, but yeah - good catch.